recipes/metis/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.0.3" %}
+
+package:
+  name: metis
+  version: {{ version }}
+
+source:
+  fn: metis-{{ version }}.tar.gz
+  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD/metis-{{ version }}.tar.gz
+  sha256: 5efa35de80703c1b2c4d0de080fafbcf4e0d363a21149a1ad2f96e0144841a55
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+
+test:
+  commands:
+    # Check for library.
+    - test -f "${PREFIX}/bin/libmetis.a"
+
+    # Check for executables.
+    - test -f "${PREFIX}/bin/graphchk"
+    - test -f "${PREFIX}/bin/kmetis"
+    - test -f "${PREFIX}/bin/mesh2dual"
+    - test -f "${PREFIX}/bin/mesh2nodal"
+    - test -f "${PREFIX}/bin/oemetis"
+    - test -f "${PREFIX}/bin/onmetis"
+    - test -f "${PREFIX}/bin/partdmesh"
+    - test -f "${PREFIX}/bin/partnmesh"
+    - test -f "${PREFIX}/bin/pmetis"
+
+    # Try running executables.
+    - graphchk
+    - kmetis
+    - mesh2dual
+    - mesh2nodal
+    - oemetis
+    - onmetis
+    - partdmesh
+    - partnmesh
+    - pmetis
+
+about:
+  home: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview
+  license: ???
```

Comment:
I can't seem to find a license for 4.0.x. I do see a license for 5.0.3, but it is described as a changed license. There does not appear to be a license in the source. Help figuring this out would be appreciated.